ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -166,8 +177,14 @@ class ConfirmPaymentMethodScreen extends React.Component<Props, never> {
                 {I18n.t("payment.changePsp")}
               </Text>
             </TouchableDefaultOpacity>
+            <View spacer={true} large={true} />
+            <TouchableDefaultOpacity
+              accessibilityLabel="why-a-fee"
```

Comment:
```suggestion
              testID="why-a-fee"
```
We should use `testID` because the `accessibilityLabel` is used to set a custom text that should be read by the screen reader for blind people.

Reply:
I can switch to test id  if you think it's better, but in my opinion that touchable should have accessibility text set. Obviously the text can't be "why-a-fee"